R/derive_param_computed.R

```diff
@@ -293,14 +293,6 @@ derive_param_computed <- function(dataset = NULL,
                                   constant_parameters = NULL,
                                   keep_nas = FALSE) {
   assert_vars(by_vars)
-  if (!missing(analysis_var)) {
-    deprecate_warn(
-      "0.12.0",
-      "derive_param_computed(analysis_var = )",
-      "derive_param_computed(set_values_to = )"
-    )
-  }
-  analysis_var <- assert_symbol(enexpr(analysis_var))
```

Comment:
I think we should not remove this part because otherwise the following does not issue a deprecation warning.
```
> derive_param_computed(
+   advs,
+   by_vars = exprs(USUBJID, VISIT),
+   parameters = c("SYSBP", "DIABP"),
+   analysis_var = CHR,
+   set_values_to = exprs(
+     AVAL = (AVAL.SYSBP + 2 * AVAL.DIABP) / 3,
+     PARAMCD = "MAP",
+     PARAM = "Mean Arterial Pressure (mmHg)",
+     AVALU = "mmHg"
+   )
+ )
```

Reply:
I was a bit confused by analysis_var and analysis_value, but I think I fixed it now, hopefully!

Reply:
Have you committed your fix?

Reply:
Will be up in a minute, still running check()